templates/Dataland/datasetCitation.html.twig

```diff
@@ -1,4 +1,4 @@
-{{ dataset.Authors }} ({{ dataset.AcceptedDate|date("Y") }}) {{ dataset.title }}. Distributed by: Gulf of Mexico Research Initiative Information and Data Cooperative (GRIIDC), Harte Research Institute, Texas A&M University-Corpus Christi. 
+{{ dataset.Authors }} ({{ dataset.AcceptedDate|date("Y") }}) {{ dataset.title }}. Distributed by: GRIIDC, Harte Research Institute, Texas A&M University-Corpus Christi.
```

Comment:
Do we need to escape the & character in this context? &amp;

Reply:
Thats how it is on prod, do we want t o change it?

Reply:
Leave it alone I say.

Reply:
ok, if it works there all is well.